beacon_node/network/src/metrics.rs

```diff
@@ -218,13 +218,30 @@ lazy_static! {
     /*
      * Bandwidth metrics
      */
+    pub static ref INBOUND_LIBP2P_QUIC_BYTES: Result<IntGauge> =
+        try_create_int_gauge("libp2p_inbound_quic_bytes", "The inbound quic bandwidth over libp2p");
+
+    pub static ref INBOUND_LIBP2P_TCP_BYTES: Result<IntGauge> =
+        try_create_int_gauge("libp2p_inbound_tcp_bytes", "The inbound tcp bandwidth over libp2p");
```

Comment:
```suggestion
    pub static ref INBOUND_LIBP2P_QUIC_BYTES: Result<IntGauge> =
        try_create_int_gauge("libp2p_inbound_bytes", "The inbound bandwidth over libp2p");
```

Best practice would be to use a metric family for all bandwidth, differentiated by a Prometheus label, e.g. `transport="tcp|quic"`.

Reply:
yeah makes sense thanks, went ahead and also put the direction into the same metric